.env.template

```diff
@@ -39,6 +39,17 @@ AZURE_STORAGE_ACCOUNT_URL=
 AZURE_STORAGE_ACCOUNT_KEY=
 AZURE_STORAGE_CONTAINER=
 
+# Set AWS keys if using AWS instances
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_DEFAULT_REGION=
```

Comment:
Can you please put these three keys below the azure speech keys (i.e move these keys above)? Also comment before them saying these are Azure speech/translation keys.

Reply:
Should I also shift the google application credential below the AWS ?

Reply:
GCP credentials are used for both storage and language purposes right? It can be there. Also have you confirmed whether the packages are working?

Reply:
Yes, I have tested each and every package and they are working fine.

Reply:
@KaranrajM I have shifted the AWS credentials above. Please review it.

Reply:
Hello @KaranrajM 
Can we pls get the PR reviewed? We need it to be merged by 15th Sept for it to be considered of our ongoing challenge. Thanks.